lib/commands/test-init.ts

```diff
@@ -0,0 +1,77 @@
+///<reference path="../.d.ts"/>
+
+"use strict";
+
+import * as path from 'path';
+import * as util from 'util';
+import {TESTING_FRAMEWORKS} from '../constants';
+
+class TestInitCommand implements ICommand {
+	constructor(private $npm: INodePackageManager,
+		private $projectData: IProjectData,
+		private $errors: IErrors,
+		private $options: IOptions,
+		private $prompter: IPrompter,
+		private $fs: IFileSystem,
+		private $resources: IResourceLoader,
+		private $logger: ILogger) {
+	}
+
+	private frameworkDependencies:IDictionary<string[]> = {
+		jasmine: [],
+		mocha: ['chai'],
+	};
+
+	public execute(args: string[]) : IFuture<void> {
+		return (() => {
+			let projectDir = this.$projectData.projectDir;
+
+			let frameworkToInstall = this.$options.frameworkName
+				|| this.$prompter.promptForChoice('Select testing framework:', TESTING_FRAMEWORKS).wait();
+			if (TESTING_FRAMEWORKS.indexOf(frameworkToInstall) === -1) {
+				this.$errors.fail(`Unknown or unsupported unit testing framework: ${frameworkToInstall}`);
+			}
+
+			['karma', 'karma-' + frameworkToInstall, 'karma-nativescript-launcher', 'nativescript-unit-test-runner']
+			.concat(this.frameworkDependencies[frameworkToInstall].map(f => 'karma-' + f))
+			.forEach(mod => {
+				this.$npm.install(mod, projectDir, {
```

Comment:
Can't we extend the install to take an array of strings of package to install, similarly to the way the npm CLI supports it?